content/en/shift_left.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Shift Left
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Left in Shift Left refers to the left-to-right (development-to-development) workflow. Shift Left is the practice of implementing tests, security, or other development practices early in the software development cycle rather than towards the end. The actual stages in the software development lifecycle may be defined in different ways based on the team and the models used.
```

Comment:
```suggestion
Left in Shift Left refers to the left-to-right (development-to-development) workflow. Shift Left is the practice of implementing tests, security, or other development practices early in the software development cycle rather than towards the end. The actual stages in the software development lifecycle may be defined in different ways based on the team and the models used.
```
```suggestion
Left in Shift Left refers to earlier stages in a software development lifecycle. Shift Left is the practice of implementing tests, security, or other development practices early in the software development lifecycle rather than towards the end. 
```

Reply:
@iamNoah1, not sure everyone will be familiar with the left-to-right concept. Shift left does only make sense if you get that. That first sentence was actually my suggestion :) If you have a better idea, I'm open to hearing it but not sure "Left in Shift Left refers to earlier stages in a software development lifecycle" makes that clear. Once you know what left means, it makes total sense.

Reply:
I am good with either suggestion. The latest suggestion is more streamlined and straightforward, but to @CathPag's point this also makes assumptions about what folks may already know or not know. also on the prior suggestion one note is that the 'development-to-development' phrasing was a bit confusing to me. What are thoughts on combing phrasing as: `"Left in Shift Left refers to earlier stages of the left-to-right software development workflow."` and omitting that last sentence starting `"The actual stages..."`

Reply:
Getting a bit turned around by the comment threading as well, but the latest looks great to me.